geometry_msgs/PoseStamped       pose
---
bool                            success
geometry_msgs/PoseStamped[]     sp
geometry_msgs/PoseStamped[]     ing
